src/App.tsx

```diff
@@ -5,7 +5,21 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const pushMovie = (movie: Movie) => {
+    if (movies.length > 0) {
+      const imbdIdMovie = movies
```

Comment:
```suggestion
      const isMoviePresent = movies
```
Better specify that it's boolean value to make it clear